src/services/searchService.js

```diff
@@ -30,17 +157,23 @@ const urlToSearchParams = (query = '') => {
   for (let key of keys) {
     for (let type of types) {
       if (key.toLowerCase() === type) {
-        return { searchType: type, searchText: parsedObject[key] }
+        return {searchType: type, searchText: parsedObject[key]}
       }
     }
   }
 
-  return { searchType: SEARCH_TYPE.title, searchText: '' }
+  return {searchType: Object.keys(SEARCH_TYPE)[0], searchText: ''}
 }
 
 const SEARCH_TYPE = {
-  title: 'title',
-  director: 'director',
+  movie: {
+    label: 'movie',
+    subpath: 'movie'
+  },
+  tvShow: {
+    label: 'TV show',
+    subpath: 'tv'
+  },
 }
 
-export { SEARCH_TYPE, paramsToUrl, urlToSearchParams, search }
+export {SEARCH_TYPE, paramsToUrl, urlToSearchParams, search, getDetails, getCollection}
```

Comment:
No need to export the object - each function/constant could be exported directly